percona-packaging/scripts/psmdb_builder.sh

```diff
@@ -0,0 +1,796 @@
+#!/bin/sh
+
+shell_quote_string() {
+  echo "$1" | sed -e 's,\([^a-zA-Z0-9/_.=-]\),\\\1,g'
+}
+
+usage () {
+    cat <<EOF
+Usage: $0 [OPTIONS]
+    The following options may be given :
+        --builddir=DIR      Absolute path to the dir where all actions will be performed
+        --get_sources       Source will be downloaded from github
+        --build_src_rpm     If it is set - src rpm will be built
+        --build_source_deb  If it is set - source deb package will be built
```

Comment:
Inconsistency in option naming? build_src_rpm vs build_source_deb

Reply:
it is the same for 3.2 and 3.4 scripts. Do you think it is better to have build_src_xxx?